topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -323,20 +323,20 @@ The resulting `concatenated_scores` needs to be re-sorted according to the align
 
 For pharmacophore-based protein target prediction, you can choose to use Galaxy tools separately and in succession as described above, or alternatively use the one-step Zauberkugel workflow as described below (Figure 3).
 
-![Snapshot of Zauberkugel workflow]({% link topics/computational-chemistry/images/zauberkugel.png %} "Zauberkugel — protein target prediction of a bioactive ligand with Align-it and ePharmaLib")
-
 > ### {% icon hands_on %} Upload the Zauberkugel workflow
 >
->    {% snippet faqs/galaxy/workflows_import.md %}
```

Comment:
I'm 80% sure the issue was too much leading space, that should work.

Reply:
Edit: ah, yes, I see you've done that below and good, that works then!

Reply:
Yes, I assume so as well.